specs/src/specs/public_key_cryptography.md

```diff
@@ -0,0 +1,66 @@
+# Public-Key Cryptography
+
+<!-- toc -->
+
+Consensus-critical data is authenticated using [ECDSA](https://www.secg.org/sec1-v2.pdf) with the curves: [Secp256k1](https://en.bitcoin.it/wiki/Secp256k1) or [Ed25519](https://en.wikipedia.org/wiki/EdDSA#Ed25519).
+
+## Secp256k1
+
+The Secp256k1 key type is used by accounts that submit transactions to be included in Celestia.
```

Comment:
Doesn't the SDK support multiple key types? Not just Secp256k1

Reply:
Yes it looks like it does. I can't seem to create one via celestia-appd though:

```go
$ ./build/celestia-appd keys add test --algo ed25519
Error: provided algorithm "ed25519" is not supported
```

so still investigating.

Ref:
1. https://docs.cosmos.network/v0.46/run-node/keyring.html#adding-keys-to-the-keyring

Reply:
Relevant: https://github.com/cosmos/cosmos-sdk/discussions/8543

Reply:
Resolving for now b/c I don't think we can actually use ed25519 keys in Cosmos SDK.